include/cpp_lmdb/transactions.hpp

```diff
@@ -38,6 +38,8 @@ class transaction {
 
     using ro_view
         = db_view<ro_iterator<key_trait, value_trait, LmdbApi>, LmdbApi>;
+    using ro_lo_view
+        = db_dup_view<ro_iterator<key_trait, value_trait, LmdbApi>, LmdbApi>;
```

Comment:
I think it makes sense to rename `db_dup_view` into something like `db_view_with_key` as it is now used not only in key duplicate scenarios